zcash_primitives/src/transaction/mod.rs

```diff
@@ -48,6 +49,41 @@ pub struct TransactionData {
     pub binding_sig: Option<Signature>,
 }
 
+impl std::fmt::Debug for TransactionData {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> Result<(), std::fmt::Error> {
+        write!(
```

Comment:
Alternatively could be written in the form:

```rust
write!(
    f,
    "TransactionData(
        overwintered: {overwintered:?},
        version: {version:?})",
    overwintered=self.overwintered,
    version=self.version,
)
```

but because each field is used only once in the format, I don't think this needs to be changed from how it currently is.

Reply:
Ah, I hadn't stumbled on this form yet. I'll keep it in mind for future changes.